src/nvme/tree.h

```diff
@@ -59,7 +59,7 @@ typedef bool (*nvme_scan_filter_t)(nvme_subsystem_t);
  * nvme_first_host() -
  * @r:
  *
- * Return: 
```

Comment:
Could you do a seperate patch for the whitespace fixes? Kind of hides the nvme_init_ctrl() move. Thanks!

Reply:
Will do. I forgot that my editor was set to delete trailing spaces on save. ;-)